### PR TITLE
[DE] Add `Kudosu`

### PR DESCRIPTION
**Pull request by Bloeckchengrafik**
Just thought would be cool

### CODE REVIEW

**Comment by Bloeckchengrafik** [comment]
fixed a grammar mistake

File: wiki/Modding/Kudosu/de.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Eine aktuelle Bestenliste findet man [hier](https://osu.ppy.sh/p/kudosu).
+
+<!-- TODO: forum modding, star priority? stubs-->
+
+<!-- TODO:
+  - history in general / forum modding usage
+  - "You could explain kudosu's usage in evaluating BN applications, its past usage in BN Ranking systems" -jbhyperion
+  - how it looks and works in beatmap discussions
+  - when is it appropriate to award kd
+->
```
Comment (TPGPL) [inline wiki/Modding/Kudosu/de.md:18]: should be removed

L9 because its not available in the English article

File: wiki/Modding/Kudosu/de.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Kudosu
```
Comment (TPGPL) [inline wiki/Modding/Kudosu/de.md:1]:
```suggestion
---
stub: true
no_native_review: true
---

# Kudosu
```

**Review by TPGPL — CHANGES_REQUESTED** [review]
also you should check this box to allow maintainers to edit your PRs

image

**Review by TPGPL — APPROVED** [review]

**Pull request merged**